js/error.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+window.error = (function () {
```

Comment:
the Idea is that it's not a error module. 
It's a message handling module that adds and removes message. 
I think you need to add a success message here as well that handles successTemplate creation.

Reply:
done